l1-contracts/hardhat.config.ts

```diff
@@ -93,7 +93,6 @@ export default {
       return {
         ...systemParams,
         ...defs,
-        VALIDIUM_MODE: process.env.VALIDIUM_MODE == "true",
```

Comment:
IMO, we should leave `VALIDIUM_MODE` as env variable, or are there any reasons to move it as input for deployment scripts?

Reply:
Yep, this variable is not used by the core monorepo, so it was an unexpected orphan variable there. Which is why we asked to replace it with a CLI flag.